pallets/gear/src/manager.rs

```diff
@@ -120,7 +120,7 @@ where
             state: common::ProgramState::Uninitialized { message_id },
         };
 
-        common::set_program_and_pages_data(program_id.into_origin(), program, Default::default());
+        common::set_program(program_id.into_origin(), program);
```

Comment:
Are we ok with fact that function can panic?

Reply:
I think no. But it's not the problem of current PR.